starknet_devnet/devnet_config.py

```diff
@@ -140,6 +141,13 @@ def parse_args(raw_args: List[str]):
         help="Specify the gas price in wei per gas unit; "
         + f"defaults to {DEFAULT_GAS_PRICE:g}",
     )
+    parser.add_argument(
+        "--timeout",
+        "-t",
+        action=NonNegativeAction,
+        default=DEFAULT_TIMEOUT,
+        help="Specify timeout for the server"
```

Comment:
State the default value in the help string (see other arg params for reference)

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/256/commits/b715d1628e248a90cc36c49fd6aa78c9d117f1fb